### ISSUE TITLE
Build error

### ISSUE THREAD

**Issue by msageryd** [blackbeam/poppler-simple#38]
I'm trying to install poppler-simple on Mac OS Catalina (10.15.7) with Node 12.21 (via NVM).

`npm i poppler-simple@latest`
gives me the following error. What should I do to dig deeper into this?

```
> poppler-simple@0.10.0 install /Users/michaelsageryd/dev/plantrail/client/pdf2jpg/node_modules/poppler-simple
> (node-gyp rebuild) || (exit 1)

  CXX(target) Release/obj.target/poppler/src/poppler.o
  CXX(target) Release/obj.target/poppler/src/NodePopplerDocument.o
../src/NodePopplerDocument.cc:60:11: error: assigning to 'PDFDoc *' from incompatible type 'std::unique_ptr<PDFDoc>'
    doc = PDFDocFactory().createPDFDoc(*fileNameA, ownerPassword, userPassword);
          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
1 error generated.
make: *** [Release/obj.target/poppler/src/NodePopplerDocument.o] Error 1
gyp ERR! build error 
gyp ERR! stack Error: `make` failed with exit code: 2
```

**Comment by blackbeam** [comment]
Hi.
I believe this is due to a breaking change introduced in popper 21.03. I'll look into it.

**Comment by blackbeam** [comment]
Fixed in [v0.10.1](https://github.com/blackbeam/poppler-simple/releases/tag/v0.10.1).

**Comment by msageryd** [comment]
Thanks.
I can confirm that it works.
Really speedy response.
